src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/CohortController.cs

```diff
@@ -226,19 +225,46 @@ public async Task<IActionResult> AddDraftApprenticeshipOrRoute(string changeCour
             if (await HasStartDateOverlap(model))
             {
                 StoreDraftApprenticeshipState(model);
-                return RedirectToAction(nameof(DraftApprenticeshipOverlapOptions));
+                return RedirectToAction(nameof(DraftApprenticeshipOverlapAlert));
             }
 
             return await SaveDraftApprenticeship(model);
         }
 
-          
+        [HttpGet]
+        [Route("add/apprenticeship/overlap-alert", Name = RouteNames.CohortOverlapAlert)]
+        public IActionResult DraftApprenticeshipOverlapAlert(DraftApprenticeshipOverlapAlertRequest request, [FromServices] IFeatureTogglesService<ProviderFeatureToggle> featureTogglesService)
+        {
+            var featureToggleEnabled = featureTogglesService.GetFeatureToggle(ProviderFeature.OverlappingTrainingDate).IsEnabled;
+            var model = PeekStoredDraftApprenticeshipState();
+
+            var vm = new DraftApprenticeshipOverlapAlertViewModel
+            {
+                OverlappingTrainingDateRequestToggleEnabled = featureToggleEnabled,
```

Comment:
this property of the model isn't used and can be removed. As can the variable above and the injected feature toggle service

Reply:
updated